requirements.txt

```diff
@@ -1,8 +1,9 @@
-scikit-learn>=0.20
+scikit-learn==0.20.4
```

Comment:
why ? aikit should work with every scikit-learn version above 0.20 (and several versions are tested)

Reply:
Since the 0.23 version of sciait-learn, several functions used in aikit are deprecated. It's just a temporary fix.

Reply:
In that case better put `sciait-learn>=0.20,<0.23` to be less constrained.